react-delegationdashboard/src/pages/Ledger/AddressTable.tsx

```diff
@@ -4,7 +4,6 @@ import { useContext, useDispatch } from 'context';
 import { HWProvider } from '@elrondnetwork/erdjs/out';
```

Comment:
maybe don't use `/out` . Can you also check in the entire project for `/out` and try to replace it if applicable?